src/home_work2/task1.js

```diff
@@ -0,0 +1,73 @@
+// Imports 
+import * as Joi from '@hapi/joi';
+import report from "../utils/status";
+const express = require('express');
```

Comment:
See, there are 2 types of module systems in JS nowadays: CommonJS (uses "require" and "module.exports") and ES6 Imports (uses "import" and "export").
CommonJS is older and used in Node.js (at least, until recently), and ES6 Imports - in browsers.
Therefore, consider not mixing them and using only one.